### PR TITLE
Fix worker leak for background blur and replacement (#2017)

### PR DESCRIPTION
**Pull request by richhx**
**Issue #:** #2017

**Description of changes:**
This fixes a worker leak for background blur and replacement by changing the `destroy` message to `stop`. Previously, when the wrong message was sent, the worker would not be properly shut down and would still run, thereby consuming unnecessary resources. Same as #2020 but not forked.

This change is for the 3.x release branch. A separate one will be created to hotfix this in 2.x.

**Testing:**
Added a unit test to `BackgroundBlurProcessor.test.ts` and verified the reproduction case as mentioned in the initial issue.

To verify the changes when using background blur in the SDK, toggling background blur on and off (e.g. calling `BackgroundBlurVideoFrameProcessor.create()` and then `processor.destroy()`) should not leak web workers anymore. This is verified by running the demo application (or the simple demo linked in the issue), switching between background blur 10% and 20%, and using the chrome task manager to see that the number of workers keeps increasing without getting deleted.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
Yes. 
1. Launch the demo application.
2. Create a meeting
3. Enable your camera
4. Join the meeting
5. Select video on by clicking the video icon.
6. Select Background Blur 10% by clicking the circle half filled white and black.
7. Select Background Blur 20%.
8. Keep alternating between 10% and 20% and notice that the number of "Dedicated Worker:"s keep increasing in the Chrome task manager (sort by CPU).

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No.

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Yes.

### CODE REVIEW

**Comment by michhyun1** [comment]
Looks like the unit test code coverage is not 100%

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "doc": "ruby script/generate-docs",
     "postdoc": "node script/update-typedoc-link.js",
     "test": "npm run test:fast",
-    "test:glob": "cross-env TS_NODE_PROJECT=test/tsconfig.json TS_NODE_CACHE=false nyc mocha --stack-trace-limit=1000  --async-stack-traces -v -r esm -rv ts-node/register -- ",
+    "test:glob": "cross-env TS_NODE_PROJECT=test/tsconfig.json TS_NODE_CACHE=false nyc --reporter=text mocha --stack-trace-limit=1000  --async-stack-traces -v -r esm -rv ts-node/register -- ",
```
Comment (richhx) [inline package.json:37]: Added this to make it easier to detect lack of coverage problems.

**Review by michhyun1 — APPROVED** [review]

**Comment by cgandhi22** [comment]
Here in testing steps, post fix, when you keep alternating between 10% and 20% and notice that the number of "Dedicated Worker:"s should not keep increasing in the Chrome task manager (sort by CPU). Correct?

**Comment by richhx** [comment]
Yes post fix, you should not see the number of workers increasing. I verified this in my own serverless demo too.

**Pull request merged**